lib/airbrake-ruby/config.rb

```diff
@@ -125,6 +125,23 @@ def merge(config_hash)
       self
     end
 
+    ##
+    # @return [Boolean] true if the config meets the requirements, false
+    #   otherwise
+    def valid?
+      return true if ignored_environment?
+      return false unless project_id.is_a?(Integer)
+      project_key.is_a?(String) && !project_key.empty?
+    end
+
+    ##
+    # @return [Boolean] true if the config ignores current environment, false
+    #   otherwise
+    def ignored_environment?
+      return false if ignore_environments.none?
```

Comment:
This seems like an unnecessary early return. The next line should cover everything, right?